docs/kap_proposals/kap_1_external_dependencies.md

```diff
@@ -27,6 +27,7 @@ Git type may also include `ref` and `subdir` parameters as illustrated below:
   source: <git_url>
   subdir: relative/path/in/repository
   ref: <commit_hash/branch/tag>
+  fetch_always: <bool>
```

Comment:
@MatteoVoges last update here needed

Reply:
@ramaro ahh I missed this one. Done with 5b632c26316309f51764a369284d3edb4415867e

Reply:
now that was quick! Thanks 👍🏼